snafu-derive/src/lib.rs

```diff
@@ -569,19 +581,31 @@ impl syn::parse::Parse for SnafuAttributeBody {
     }
 }
 
+struct DocComment(SnafuAttribute);
+
+impl syn::parse::Parse for DocComment {
+    fn parse(input: syn::parse::ParseStream) -> SynResult<Self> {
+        let _: Token![=] = input.parse()?;
```

Comment:
It’s silly, but I’m avoiding using `Token!` macro in favor of the longer-form `use syn::token::…`;